cinema/tests/test_movie_api.py

```diff
@@ -42,21 +40,15 @@ def test_get_movies(self):
             self.assertEqual(movies.data[0][field], titanic[field])
```

Comment:
In `test_get_movies`, the test assumes the order of fields returned by the API. Consider using more flexible assertions or explicitly checking each field to avoid test failures due to unexpected API changes.